server/models/db/schema.ts

```diff
@@ -1,8 +1,106 @@
+import { unique } from 'drizzle-orm/mysql-core';
 import { sqliteTable } from 'drizzle-orm/sqlite-core';
 import * as t from 'drizzle-orm/sqlite-core';
 
+// TODO move this into types.ts
+const POST_TYPES = ['question', 'answer'] as const;
+export type PostType = (typeof POST_TYPES)[number];
```

Comment:
We should probably change this to POSTTYPES unless snake case is allowed for constants

Reply:
iirc javascript constants are in snake case, since otherwise it could be hard to distinguish words (e.g. DATBASESCHEMATYPES or something)